internal/providersdkv2/resource_vault_cluster_test.go

```diff
@@ -295,17 +298,27 @@ func updateClusterTier(t *testing.T, in *inputT) resource.TestStep {
 	}
 }
 
-// This step verifies the successful update of "public_endpoint", "proxy_endpoint", "audit_log", "metrics" and MVU config.
+// This step verifies the successful update of "public_endpoint", "proxy_endpoint", "ip_allowlist", "audit_log", "metrics" and MVU config.
 func updatePublicProxyObservabilityAndMVU(t *testing.T, in *inputT) resource.TestStep {
```

Comment:
Nit: maybe we can rename this from `updatePublicProxyObservabilityAndMVU` to `updateNetworkObservabilityAndMVU` where `Network` captures  all three of public, proxy, and IP allowlist because the current name isn't really accurate anymore

Reply:
Agree, renamed it!